src/search.js

```diff
@@ -51,7 +50,14 @@ var Search = React.createClass({
     const idealSearchBoxWidth = Math.max(17, this.state.terms && this.state.terms.length*1.1 || 0)
     const formProps = universal ? {action: "/search/", method: "get"} : {action: ''}
     const simpleSearchBox = <div className='search-wrapper' style={{width: idealSearchBoxWidth + 'em'}}>
-      <form {...formProps}><input className='search-input' type="search"
+      <form {...formProps} role="search">
```

Comment:
Adding role of search here to help identify what this form is being used for

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role